clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/KubernetesUtil.groovy

```diff
@@ -26,6 +27,7 @@ class KubernetesUtil {
   static String SECURITY_GROUP_LABEL_PREFIX = "security-group-"
   static String LOAD_BALANCER_LABEL_PREFIX = "load-balancer-"
   static String REPLICATION_CONTROLLER_LABEL = "replication-controller"
+  static String DEFAULT_REGISTRY = "gcr.io"
```

Comment:
Feels like having the option to specify what default is used when the registry is omitted from a request is reasonable. And the default default (heh) can still be `gcr.io`.